tests/data_association/test_data_association.py

```diff
@@ -299,7 +306,11 @@ def test_create_computation_graph(self):
         )
         da = DataAssociation(min_track_len=3, triangulation_options=triangulation_options)
         expected_sfm_data, expected_metrics = da.run(
-            len(cameras), cameras, corr_idxs_graph, keypoints_list, cameras_gt=cameras_gt, relative_pose_priors={}
+            len(cameras),
```

Comment:
nit: can we make these all keyword args, since I have a hard time remembering what represents what?